website/blog/2021-06-28-release-0-2.md

```diff
@@ -0,0 +1,43 @@
+---
+slug: release-sb
+title: Releasing SuperBench 0.2.0
+author: Tingting Qin
+author_title: SuperBench Team
+author_url: https://github.com/TobeyQin
+# author_image_url: https://avatars.githubusercontent.com/u/6807041?v=4
+tags: [superbench, announcement, release]
+---
+
+We are very happy to announce that **SuperBench 0.2.0 version** is officially released today! 
+
+You can install and try superbench by following [Getting Started Tutorial](https://microsoft.github.io/superbenchmark/docs/getting-started/installation).
+
+## SuperBench 0.2.0 Release Notes
+
+### SuperBench Framework
+
+* Implemented SuperBench CLI to provide a command line interface.
+* Implemented SuperBench Runner.
+* Implemented SuperBench Executor.
+* Implemented SuperBench Benchmark framework.
+
+### Supported Benchmarks
+
+* Supported Micro-benchmarks
+  * GEMM FLOPS (GFLOPS, TensorCore, cuBLAS, cuDNN)
+  * Kernel Launch Time (Kernel_Launch_Event_Time, Kernel_Launch_Wall_Time)
+  * Operator Performance (MatMul, Sharding_MatMul)
+* Supoorted Model-benchmarks
+  * CNN models
+    * ResNet (ResNet-50, ResNet-101, ResNet-152)
+    * DenseNet (DenseNet-169, DenseNet-201)
+    * VGG (VGG-11, VGG-13, VGG-16, VGG-19)
```

Comment:
We have more CNN model actually. @guoshzhao any suggestions?

Reply:
For supported models in the release page, I think we can list all of them

Reply:
add "..."